src/sbt-test/universal/multiproject-classifiers/build.sbt

```diff
@@ -0,0 +1,42 @@
+lazy val appVersion = "1.0"
+
+lazy val mySettings: Seq[Setting[_]] =
+  Seq(organization := "org.test", version := appVersion, TaskKey[Unit]("showFiles") := {
+    System.out.synchronized {
+      println("Files in [" + name.value + "]")
+      val files = (target.value / "universal/stage").**(AllPassFilter).get
+      files foreach println
+    }
+  })
+
+lazy val assets = config("assets")
+
+lazy val sub = project
+  .in(file("sub"))
+  .enablePlugins(JavaAppPackaging)
+  .settings(mySettings)
+  .settings(
+    ivyConfigurations += assets,
+    artifact in assets := artifact.value.withClassifier(classifier = Some("assets")),
```

Comment:
It's generated from https://github.com/sbt/librarymanagement/blob/1.0/librarymanagement/src/main/contraband/librarymanagement.json. I should make sure the generated code goes into GitHub like other sbt modules.